### DISCUSSION TITLE
responsive borders?

### DISCUSSION THREAD

**Discussion by Hyp3rionM4x** [twbs/bootstrap#33560] [Ideas]
Hello Bootstrap thinkers.

I was the other day working on a project of my own and i happened to come across an idea where it was kinda hard for me to find a solution (found one temporary tho).

So i thought if possible in future you could add it.

Just like all the other cool second-hand utilities we can use, can u add border to be responsive on "sm,md,lg,etc..."? Would help alot! Like when to add/remove these (add/subtrack, etc)!

Thanks a lot! :)

**Comment by wnunezc** [comment]
if there is something, it is the border size but this refers to the radius or curve of the corners ... we would have to create the appropriate classes and see what the basic measurements would be based on ...

**Reply by Hyp3rionM4x** [reply]
Hmm, i get where you are coming from but in the situation i am refering to at the moment are really just to see with the help from css media queries that if the parent (the element the borders are really being used upon), are on a specific view-port size, then it would change based on this. In my situation, i really just need some sort of way to say (if on a less x-pixel screen, border:none/0;, else border:1px;, or something similar). Does this makes sense?

**Reply by wnunezc** [reply]
Do you know that the behavior of this "sm,md,lg,etc..." is based on media queries?

BS does it or you do it is the same method, the difference is that BS first uses scss where it would declare variables etc and when compiling, the behavior you are looking for would appear within the media queries ...

https://getbootstrap.com/docs/5.0/utilities/api/

**Comment by dvmuccillo** [comment]
Hi,
if you are going for v5 i believe you can do this with the [Utility API](https://getbootstrap.com/docs/5.0/utilities/api/).
